ddl/db_integration_test.go

```diff
@@ -1351,6 +1353,27 @@ func (s *testIntegrationSuite6) TestCreateTableTooLarge(c *C) {
 	atomic.StoreUint32(&ddl.TableColumnCountLimit, originLimit)
 }
 
+func (s *testIntegrationSuite8) TestCreateTableTooManyIndexes(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+
+	count := int(atomic.LoadUint32(&ddl.TableIndexCountLimit) + 1)
```

Comment:
If you use the global variable, the test should be serial.

Reply:
I removed the global variable
PTAL @wjhuang2016